src/plugins/memory/memory.yml

```diff
@@ -1,3 +1,12 @@
 name:
   en: Memory
   de: Memory
+tilePairs:
+  en: Tile pairs
+  de: Kachelpaare
```

Comment:
Just a thought: Now that I see the German translation I wonder if we shouldn't go with the term "card" instead of "tile" as it would resemble an analog game more closely. Even if we want to stick to "tile" in code, I'd suggest to use the words "card"/"Karte" on the UI here.

Reply:
Changed the translations as you suggested 👍